### ISSUE TITLE
Feature Request: Full reset of cached calculated values

### ISSUE THREAD

**Issue by jochenwezel** [EPPlusSoftware/EPPlus#97]
Since the calculation engine is not implemented 100% compatible to MS Excel (at least not yet, many important functions are missing), I need to disable the calculation engine in EPPlus and after all changes, open with MS Excel and do a full recalculation there.

If a formula is calculated in EPPlus already, the EPPlus calculation engines saves an internal result value for that cell.

Since MS Excel doesn't realize that the cached value might already be obsolete (e.g. due to changed values of depending cells), MS Excel calculates with the old, obsolete cached values, resulting in wrong results.

For solving this problem, I'd like to reset the cached values of all cells with formulas in EPPlus.
For this, I implemented following method in EPPlus\ExcelRangeBase.cs using EPPlus 4.5.3.

```C#
        public static void ClearCachedCalculatedFormulaValue(ExcelRangeBase range, int row, int col)
        {
            if (range._worksheet.GetFormula(row, col) != null)
            {
                range._worksheet.SetValueInner(row, col, null);
            }
        }

        public void ClearCachedCalculatedFormulaValue()
        {
            ClearCachedCalculatedFormulaValue(this, _fromRow, _fromCol);
        }
```

Now I just have to call this method for every cell in every worksheet.

I'd like to ask you for adding this feature into your current version.

**Comment by JanKallman** [comment]
Yes, this is something we can look into for the next version. EPPlus 5.1.1 has 39 new functions, so you should get a better functionality there. The long term goal is to improve the formula parser and implement more functions.

**Comment by swmal** [comment]
@jochenwezel As Jan mentioned we have added 39 new functions in EPPlus 5. See this list: 

https://www.epplussoftware.com/Developers/FormulaCalc

We will continue to add more formulas over time.

**Comment by jochenwezel** [comment]
@JanKallman @swmal 
Yes, I already realized that you added many functions. But still the job is not yet done. And even if functions are available to 100%, compatibility might be < 100%.

Additionally, there might be some special situations when I really want to reset these calculation caches; with MS Excel Interop it's not possible, EPPlus would get an advantage here ;-)

**Comment by swmal** [comment]
@jochenwezel We will look into adding this functionality. And the job with the functons continues ;)

**Comment by swmal** [comment]
Included in EPPlus 5.1.2
